server/src/main/kotlin/suwayomi/tachidesk/manga/controller/UpdateController.kt

```diff
@@ -82,8 +108,21 @@ object UpdateController {
         }
     }
 
-    fun updateSummary(ctx: Context) {
+    val updateSummary = documented(
+        document()
+            .operation {
+                it.summary("Get the current update summary")
+                it.description(
+                    """
+                Will return the current update summary.
+                Also will retain the past update summary till the next update.
+                """.trimIndent()
+                )
+            }
+            .result<UpdateStatusSummary>("OK", "application/json")
+    ) { ctx ->
         val updater by DI.global.instance<IUpdater>()
-        ctx.json(updater.getStatus().value.getJsonSummary())
+        ctx.json(updater.getStatus().value.getSummary())
```

Comment:
don't make the json yourself, `ctx.result` will automatically convert to json when you give a non-string/non-stream object.